[PPS-169] Created Langugapack 2.0 on v14 for windows-x64

moreparesh opened the pull request:
- Build languagepack for Windows-x64 from v14 branch
Perl version 5.34
Python version 3.9
Tcl/TK version 8.6
Setuptools 57.0.0

moreparesh committed:
[PPS-169] Created Langugapack 2.0 on v14 for windows-x64

- Build languagepack for Windows-x64 from v14 branch
Perl version 5.34
Python version 3.9
Tcl/TK version 8.6
Setuptools 57.0.0

sandeep-edb reviewed (changes requested):
Paresh, can we use variables instead of hard-coding the versions in the script? Also, I don't see changes for LP variables in settings.sh.in in the patch. I guess that's also required?

moreparesh committed:
[PPS-169] Update Langugapack 2.0 changes

- Removed hardcode values for TCL and TK version
- Updated setting file with new LP installation location.

moreparesh commented:
> Paresh, can we use variables instead of hard-coding the versions in the script? Also, I don't see changes for LP variables in settings.sh.in in the patch. I guess that's also required?

I have updated version.sh with TCL/TK version, also added place holder for python batch script. LP Variables added in setting.sh.in

moreparesh committed:
[PPS-169] Update Langugapack 2.0 changes

- Updated Description about why we are using specific for TCL TK version
- remove recusrive flag to delete a file from build script.

Semab-Tariq commented:
Hi @moreparesh 

PG-14beta2 build is still failing while building with new LP and Perl-5.34 
Error is the same as mentioned in PPS-210
